bootstrap/bootstrap.go

```diff
@@ -526,10 +526,6 @@ func (b *Bootstrap) preparePlugins() error {
 }
 
 func (b *Bootstrap) validatePluginCheckout(checkout *pluginCheckout) error {
-	if !b.Config.PluginValidation {
```

Comment:
Moving this further down in the function seems like a change in behaviour since the code below it could return an `err` and break the bootstrap, specially this bit:

<img width="834" alt="image" src="https://user-images.githubusercontent.com/25882/62019857-4be55200-b1f3-11e9-9f56-8a4b14801341.png">

Was that intentional? If so, totally fine. Just wanted to double check!

Reply:
You are totally right @keithpitt, will fix.